pyproject.toml

```diff
@@ -33,7 +33,7 @@ target-version = "py311"
 
 [tool.ruff.lint]
 select = ["ALL"]
-ignore = ["D104", "D100", "D103", "D211", "D213", "D101", "D102", "D105", "D106", "D107"]
+ignore = ["D104", "D100", "D103", "D211", "D213", "D101", "D102", "D105", "D106", "D107", "D203", "ISC001", "COM812"]
```

Comment:
Can you explain those new exceptions, please? Here, in comment? Thanks!

Reply:
> Can you explain those new exceptions, please? Here, in comment? Thanks!

These linter rules are ignored because they can get in conflict with formatter. They don't seem crucial.

I'm also ignoring them to supress `ruff format` warnings.

Exact rules:
https://docs.astral.sh/ruff/rules/single-line-implicit-string-concatenation/
https://docs.astral.sh/ruff/rules/prohibited-trailing-comma/

Ruff docs on linter vs formatter conflicts:
https://docs.astral.sh/ruff/formatter/#conflicting-lint-rules